CppCoreGuidelines.md

```diff
@@ -12144,23 +12144,23 @@ References are never owners.
 
 The names are mostly ISO standard-library style (lower case and underscore):
 
-* `T*`			// The `T*` is not an owner, may be `nullptr` (Assumed to be pointing to a single element)
-* `char*`		// A C-style string (a zero-terminated array of characters); can be `nullptr`
-* `const char*`	// A C-style string; can be `nullptr`
-* `T&`			// The `T&` is not an owner, may not be `&(T&)*nullptr` (language rule)
+* `T*`			// The `T*` is not an owner, may be null; assumed to be pointing to a single element.
+* `char*`		// A C-style string (a zero-terminated array of characters); may be null.
+* `const char*` 	// A C-style string; may be null.
+* `T&`			// The `T&` is not an owner, must not be `*static_cast<T*>(nullptr)` (language rule; there are no "null references").
```

Comment:
Are you referring to the line with `T&`? You cannot say "not null", because there is no such thing as a "null reference": You cannot restrict something to not be something that doesn't exist. I think the original approach is OK here (though it was misspelled), and the term "null reference" is used as a colloquialism (hence in quotes).